client/src/browser-table.html

```diff
@@ -252,10 +252,7 @@ <h1 label>Browser support</h1>
         <a title="Stable" href="https://www.chromestatus.com/feature/4696261944934400"></a>
         <a title="Stable" href="https://www.chromestatus.com/feature/4696261944934400"></a>
         <a title="Stable" href="https://webkit.org/status/#feature-custom-elements"></a>
-        <a href="https://platform-status.mozilla.org/#custom-elements">
-          <div title="Polyfill"></div>
-          <div title="Developing"></div>
-        </a>
+        <a title="Stable" href="https://developer.mozilla.org/en-US/docs/Mozilla/Firefox/Releases/63"></a>
```

Comment:
This should’ve remained as: https://platform-status.mozilla.org/#custom-elements
```suggestion
        <a title="Stable" href="https://platform-status.mozilla.org/#custom-elements"></a>
```